src/scss/sections/get-started/get-started.scss

```diff
@@ -0,0 +1,75 @@
+.get-started {
+	padding: 44px 0 30px;
+	background: $bg-stat-dark;
+
+	@media all and (min-width: $md) {
+		padding: 55px 0 56px;
+	}
+
+	.container {
+		max-width: 1297px;
+		background-image: url( '/img/get-started/clouds.png') no-repeat;
```

Comment:
Strange choice of positioning bg image. You can add normal inner block inside container, which will contains all other nested elements, and set this bg to it. Or you can set this bg to the center of the section with auto or percentage dimensions. Container is like helper/worker class, it doesn't need to be styled somehow.